annotations/admin.py

```diff
@@ -36,10 +36,21 @@ class VogonUserAdmin(UserAdmin):
 class TextAdmin(admin.ModelAdmin):
     list_display = ('uri', 'title', 'created')
 
+class RelationAdmin(admin.ModelAdmin):
+    list_display = ('ID', 'OCCURS_IN', 'CREATED_BY', 'CREATED_AT')
+    def ID(self, obj):
```

Comment:
@erhkamal319 Great start! I wonder, though, why you define all of these methods on the RelationAdmin? For most of these, you can simply use the fields on the model itself (almost always) -- e.g. ``'id', 'created'``. For ForeignKey fields (e.g. ``occursIn``, ``createdBy``), try adding those directly, too: if I'm not mistaken, Django will evaluate the ``__unicode__()`` method on the target model, and return that as the value to display. Give it a try!

Oh, and try to avoid all-caps attributes/methods on classes. They only place we really use all-caps on class attributes is for constants like choices for a CharField.